packages/component-library/src/CardDetail/ExploreRelated.js

```diff
@@ -0,0 +1,111 @@
+/** @jsx jsx */
+import { jsx, css } from "@emotion/core";
+import PropTypes from "prop-types";
+import shortid from "shortid";
+import { Link } from "react-router";
+import { ThemeProvider } from "@material-ui/styles";
+
+import { MaterialTheme } from "../_Themes/index";
+import { PageLayout } from "../..";
+import { CivicCardLayoutPreviewTitleOnly } from "../index";
+
+const sectionMarginSmall = css`
+  display: block;
+  margin: 12px auto;
+`;
+
+const sectionMaxWidthSmall = css`
+  max-width: 900px;
+`;
+
+const listContainerStyle = css`
+  width: 100%;
+  display: flex;
+  padding: 0;
+  justify-content: space-evenly;
+  flex-wrap: wrap;
+  box-sizing: border-box;
+`;
+
+const listItemStyle = css`
+  width: 45%;
+  margin: 5px;
+  box-sizing: border-box;
+  list-style-type: none;
+`;
+
+const matchRelatedCardsByTags = (slug, baseTags, entries) => {
+  const relatedCards = entries
+    .map(entry => {
+      return {
+        numOfSimilarTags: entry.component.tags
+          ? entry.component.tags.filter(tag => {
+              return baseTags.includes(tag);
+            }).length / entry.component.tags.length
```

Comment:
remove / entry.component.tags.length -- let's just use the total number of matching tags